changelogs/unreleased/add-check-for-user-input-of-the-number-of-iproto-threads.md

```diff
@@ -0,0 +1,7 @@
+## core/bugfix
+
+ * At the moment, user can set any number of iproto threads, which leads
+   to incorrect behavior if the specified number of threads is less than
+   or equal to zero or too large. Added check for user input of the number
+   of iproto threads - value must be > 0 and less then or equal to 1000
```

Comment:
Nit: for the sake of changelog, I'd leave only the last message:
Added check for user input of the number of iproto threads - value must be > 0 and less then or equal to 1000 (gh-6005).